core/src/test/resources/docker-compose-imagename-parsing-v1.yml

```diff
@@ -2,5 +2,3 @@ redis:
     image: redis
 mysql:
     image: mysql
-custom:
```

Comment:
could you please clarify why it was removed? :)

Reply:
Hmm I think I was trying to make sure the tests only tested one thing (we have other test cases and other example compose files that cover the build scenario).

Still, I now think I'll restore it to how it was, to give us better coverage for v1 compose files.